modules/rational-defaults.el

```diff
@@ -39,15 +69,15 @@
 (customize-set-variable 'kill-do-not-save-duplicates t)
 
 ;; Make scrolling less stuttered
-(setq auto-window-vscroll nil)
+(customize-set-variable auto-window-vscroll nil)
 (customize-set-variable 'fast-but-imprecise-scrolling t)
 (customize-set-variable 'scroll-conservatively 101)
 (customize-set-variable 'scroll-margin 0)
 (customize-set-variable 'scroll-preserve-screen-position t)
 
 ;; Better support for files with long lines
-(setq-default bidi-paragraph-direction 'left-to-right)
-(setq-default bidi-inhibit-bpa t)
+(customize-set-variable-default bidi-paragraph-direction 'left-to-right)
+(customize-set-variable-default bidi-inhibit-bpa t)
```

Comment:
`bidi-inhibit-bpa` is not a `defcustom` variable, so `setq-default` is more appropriate here.